operatorhub/openshift/config.yaml

```diff
@@ -40,6 +40,14 @@ image-substitutions:
       envKeys:
       # the container name for resolvers controller is "controller" :)
       - IMAGE_PIPELINES_CONTROLLER
+- image: registry.redhat.io/openshift-pipelines/pipelines-events-rhel8@
+  replaceLocations:
+    envTargets:
+      - deploymentName: openshift-pipelines-operator
+        containerName: openshift-pipelines-operator-lifecycle
+        envKeys:
+          # not sure what should be below key
+          - IMAGE_PIPELINES_ARG_EVENTS_IMAGE
```

Comment:
@piyush-garg how do we decide what should be the key here ?

Reply:
@rupalibehera https://tektoncd.slack.com/archives/CKZQUT3MH/p1681364733658579?thread_ts=1681248315.967969&cid=CKZQUT3MH
```

```